### PR TITLE
MNT: Tweak variable PV names, add gitignore/gitattributes/etc.

### PR DESCRIPTION
**Pull request by klauer**
From https://confluence.slac.stanford.edu/display/PCDS/TwinCAT+3+Git+Setup+and+Best+Practices

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,11 @@
+*.tsproj    text eol=crlf
```
Comment (teddyrendahl) [inline .gitattributes:1]: Thank you! This is what we need. Didn't see this was in the best practices

**Review by teddyrendahl — APPROVED** [review]
Looks good.

File: TC_project/TwinCAT Project1/TwinCAT Project1/Untitled1/DUTs/DUT_STRUCT_TEST.TcDUT [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
     <Declaration><![CDATA[TYPE DUT_STRUCT_TEST :
 STRUCT
 
 	{attribute 'pytmc' := '
-		pv: ADS_TST:SetfAmpRB
-		pv: ADS_TST:SetfAmp
+		pv: ADS_TST:fAmp
 		io: o
-		pv: ADS_TST:GetfAmp
+		pv: ADS_TST:ReadOnly:fAmp
```
Comment (teddyrendahl) [inline TC_project/TwinCAT Project1/TwinCAT Project1/Untitled1/DUTs/DUT_STRUCT_TEST.TcDUT:10]: I thought we decided to stick with the `_RBV` formatting? Won't pytmc automatically add these now?

Comment (klauer) [reply TC_project/TwinCAT Project1/TwinCAT Project1/Untitled1/DUTs/DUT_STRUCT_TEST.TcDUT:10]: Yes, you're right. Though this repo is labeled 'example', I think our intention was to use it more as a pytmc test project.

So: you'll notice that the above PV `ADS_TST:fAmp`, marked `io: o`, will create both `ADS_TST:fAmp` and `ADS_TST:fAmp_RBV`. The `:ReadOnly:` portion (which will generate only `ADS_TST:ReadOnly:fAmp_RBV`) will just make it easier for us to differentiate the generated `io: o` vs `io: i` PVs from the IOC.

**Pull request merged**